orchestrator-swarm/src/main/java/com/sequenceiq/cloudbreak/orchestrator/swarm/SwarmContainerOrchestrator.java

```diff
@@ -48,34 +57,45 @@
      * @return The API address of the container orchestrator
      */
     @Override
-    public void bootstrap(String gatewayAddress, Set<Node> nodes, int consulServerCount) throws CloudbreakOrchestratorException {
+    public Map<Integer, ContainerBootstrap> bootstrap(String gatewayAddress, Set<Node> nodes, int consulServerCount) throws CloudbreakOrchestratorException {
```

Comment:
I think putting the logic of handling too much nodes here is making this interface (and both the implementation) really complicated. This logic can be outside in the bootstrapCluster method, the interface won't have to change, return types can be void, maybe the ContainerOrchestrator can have a simple method like getMaxBootstrapNodes that would return the MAX_IP_FOR_ONE_REQUEST constant in this case.
Please discuss it with @keyki or @akanto